lib/raven/client.rb

```diff
@@ -104,11 +104,7 @@ def generate_auth_header
     end
 
     def strict_encode64(string)
-      if Base64.respond_to? :strict_encode64
-        Base64.strict_encode64 string
-      else # Ruby 1.8
-        Base64.encode64(string)[0..-2]
-      end
+      Base64.strict_encode64(string) if Base64.respond_to?(:strict_encode64)
```

Comment:
It's defined in 1.9.3:
http://ruby-doc.org/stdlib-1.9.3/libdoc/base64/rdoc/Base64.html#method-i-strict_encode64